pkgs/top-level/all-packages.nix

```diff
@@ -10401,8 +10401,8 @@ in
   aws-adfs = with python3Packages; toPythonApplication aws-adfs;
 
   inherit (callPackages ../development/tools/electron { })
-    electron_3 electron_4 electron_5 electron_6 electron_7 electron_8 electron_9;
-  electron = electron_9;
+    electron_3 electron_4 electron_5 electron_6 electron_7 electron_8 electron_9 electron_10;
+  electron = electron_10;
```

Comment:
can we put this in `generic.nix`? then u can just inherit it here. I think you'll have to turn that attr set in `generic.nix` recursive with `rec` `rec { ...`

Reply:
Not sure what exactly do you mean. Moreover I don't have time to figure this out right now.

Reply:
@prusnak I pushed it https://github.com/NixOS/nixpkgs/pull/96871/commits/a079657f2041a1e7864c0a4e60812e7a72f59ccd

Reply:
Thank you!